kf_lib_data_ingest/app/cli.py

```diff
@@ -150,14 +150,10 @@ def ingest(
 ):
     """
     Run the Kids First data ingest pipeline.
-
-    \b
```

Comment:
Can you undelete this section? We actually do need it bc the CLI renders it in its help text for positional arguments